content/ecosystem/data.toml

```diff
@@ -502,7 +502,8 @@ categories = ["audio"]
 [[items]]
 name = "miniquad"
 source = "crates"
-categories = ["2drendering"]
+categories = ["3drendering"]
```

Comment:
Do you think `miniquad` could belong in both categories, or do you consider it to be primarily 3D focused? You're welcome to put it in both if you think it would fit :)

Reply:
Ok, make sense! Added both.